### PR TITLE
Uncomment plans tests + a websocket test for plans

### PR DESCRIPTION
**Pull request by mari-linhares**

### CODE REVIEW

File: syft/federated/plan.py [hunk 1]
```diff
@@ -90,9 +99,9 @@ def __init__(
 
         # Info about the plan stored
         self.plan = list()
-        self.readable_plan = list()
-        self.arg_ids = list()
-        self.result_ids = list()
+        self.readable_plan = readable_plan if readable_plan else []
```
Comment (robert-wagner) [inline syft/federated/plan.py]: Could you add an is None here. Technically everything with a false truth value would be converted to [] here

File: syft/federated/plan.py [hunk 2]
```diff
@@ -185,18 +194,43 @@ def find_location(self, args):
...
+        to_ids: List[Union[str, int]],
+        from_worker: Union[str, int] = None,
+        to_worker: Union[str, int] = None,
+    ):
         """Replaces pairs of tensor ids in the plan stored.
 
         Args:
-            from_ids: the left part of the pair: ids to change.
-            to_ids: the right part of the pair: ids to replace with.
+            from_ids: The left part of the pair: ids to change.
```
Comment (robert-wagner) [inline syft/federated/plan.py]: The right and left part of the pair are unclear. Can you clarify this morning/just remove that part

File: syft/federated/plan.py [hunk 3]
```diff
@@ -185,18 +194,43 @@ def find_location(self, args):
...
+            to_ids: The right part of the pair: ids to replace with.
+            from_worker: The previous worker that built the plan.
+            to_worker: The new worker that is running the plan.
         """
+        if not from_worker:
```
Comment (robert-wagner) [inline syft/federated/plan.py]: What if the previous worker had id 0

File: syft/federated/plan.py [hunk 4]
```diff
@@ -185,18 +194,43 @@ def find_location(self, args):
...
+            from_worker = self.id
+        if not to_worker:
```
Comment (robert-wagner) [inline syft/federated/plan.py]: Ditto

**Comment by robert-wagner** [comment]
Reviewed

File: test/federated/test_plan.py [hunk 5]
```diff
@@ -189,3 +198,33 @@ def my_plan(data):
 
     x = th.tensor([-1, 2, 3])
     assert (deserialized_plan(x) == th.tensor([-42, 24, 46])).all()
+
+
+def test_plan_execute_remotelly(hook, start_proc):
```
Comment (midokura-silvia) [inline test/federated/test_plan.py]: typo in word remotely

File: syft/federated/plan.py (hunk 2 above)
Comment (midokura-silvia) [reply syft/federated/plan.py]: I propose to remove the reference to pairs. There are two lists and no pairs. Maybe check that the lists have the same length.

**Pull request merged**